### ISSUE TITLE
White space

### ISSUE THREAD

**Issue by danperazzo** [sanowar-raihan/nerf-meta#3]
Hello, thank you for releasing this implementation. I tried to use the scripts for view synthesis from a single image. I am running with CUDA 10.1 in a Ubuntu 16.04. I am using the lamps set of images and the lamps config script. However, when I end training, the resulting video is blank, with only a white void.

**Comment by munshisanowarraihan** [comment]
Hi. The result video is blank because training did not converge. If you have an "unlucky" initialization, ReLU will only emit zero values for sigma and optimization will fail. 

The solution is to replace `F.relu()` in the line below with `F.softplus()` :
https://github.com/sanowar-raihan/nerf-meta/blob/85418c06f4868dcedc077749a21ab785065ac92b/models/nerf.py#L75

This is discussed in the [mip-NeRF](https://arxiv.org/abs/2103.13415) paper:
> In the original NeRF paper, the activation functions used by the MLP to construct the predicted density τ and color c are a ReLU and a sigmoid, respectively. Instead of a ReLU as the activation function to produce τ , we use a shifted softplus: log(1 + exp(x − 1)). We found that using a softplus yielded a smoother optimization problem that is less prone to catastrophic failure modes in which the MLP emits negative values everywhere (in which case all gradients from τ are zero and optimization will fail).

**Comment by munshisanowarraihan** [comment]
Resolved with #4. Closing.
